contrib/raftexample/raft.go

```diff
@@ -247,7 +302,76 @@ func (rc *raftNode) stopHTTP() {
 	<-rc.httpdonec
 }
 
+func (rc *raftNode) applySnapshot(snapshotToSave raftpb.Snapshot) {
+	if raft.IsEmptySnap(snapshotToSave) {
+		return
+	}
+
+	log.Printf("applying snapshot at index %d", rc.snapshotIndex)
```

Comment:
this function does not really apply the snapshot. it just sends the snapshot to the apply routine.